docs: add custom styling & custom logo (#531)
asyncapi/asyncapi-react: README.md

import "@asyncapi/react-component/styles/default.min.css";
```

> ### Custom styles
> The AsyncAPI React component does not set any global fonts and any global reset styles. This provides the option to define your custom `font-family` and other styling.

Reviewer: ```suggestion
### Custom styles
The AsyncAPI React component does not set any global fonts or reset styles. This component provides the option to define your custom `font-family` and other styling.
```

Author: `The AsyncAPI React component does not set any global fonts and any global reset styles`
The "or" would not be sufficient since it there are no global fonts sets and a number of styles are reset.

Author: I updated that part.

> The AsyncAPI React component reset a number of styles and does not set any global fonts. This component provides the option to define your custom `font-family` and other styling.

Reviewer 2: @thim81 The component doesn't reset `a number of styles` 😅

Author: I ll take it out 🤦‍♂️

> This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using AsyncAPI React component.

Reviewer 2: ```suggestion
This can be done by defining the styles in a file or inline using a `<style>` tag in the `<head>` section of the page where you are using AsyncAPI React component.
```

Example custom styles (defined in the `styles/custom.css` file):
```css
html {
  -moz-tab-size: 4;
  -o-tab-size: 4;
  tab-size: 4;
  line-height: 1.15;
  -webkit-text-size-adjust: 100%;
}

body {
  margin: 0;
  font-family: system-ui, -apple-system, Segoe UI, Roboto, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
}
```

> If you are using the component in a project that uses bundler like Webpack don't forget to import the custom styles.

Reviewer: ```suggestion
If you are using the component in a project that uses a bundler like Webpack, don't forget to import the custom styles.
```

import "@asyncapi/react-component/styles/default.min.css";
```

### Custom styles

> The asyncapi react-component does not set any global fonts. This provides the option to define your custom font-family and related font styling.
> This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using asyncapi react-component.

Reviewer 2: ```suggestion
The AsyncAPI React component does not set any global fonts and any global reset styles. This provides the option to define your custom `font-family` and other styling.

This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using AsyncAPI React component.
```

> Custom font styling (like used in AsyncAPI studio): `styles/custom.css` >>

Reviewer 2: ```suggestion
Example custom styles (used in the [AsyncAPI studio](https://github.com/asyncapi/studio)):

```

Author: I refer to the "styles/custom.css" a bit later, so perhaps we can keep it?

Reviewer 2: I thought the user would understand that this name refers to the custom styles, but you could add information that the file is called `styles/custom.css` like:

```md
Example custom styles (defined in the `styles/custom.css` file):
```

Also we don't need to add info where that style is used.

Author: Agreed, I had included the "AsyncAPI studio" style since that was the style that is used in the custom example.

I have updated the readme and removed the AsyncAPI studio part.

```css
html {
  -moz-tab-size: 4;
  -o-tab-size: 4;
  tab-size: 4;
  line-height: 1.15;
  -webkit-text-size-adjust: 100%;
}

body {
  margin: 0;
  font-family: system-ui, -apple-system, Segoe UI, Roboto, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
}
```

> If you are using the component in React or another front-end framework, don't forget to import the custom style.
> ``` js
> import "@asyncapi/react-component/styles/default.min.css";
> import "styles/custom.css";
> ```

Reviewer 2: ```suggestion
If you are using the component in project project that uses bundler like Webpack don't forget to import the custom styles.

``` js
import "styles/custom.css";
import "@asyncapi/react-component/styles/default.min.css";
```
```

Custom styles should be applied first. Css don't override styles but uses first defined. :)

> If you are using the standalone bundle, you can put the custom font styling as a style sheet link or as an inline style in the `<head>` section of the HTML code.
>   
> ```html
> <!DOCTYPE html>
> <html>
>   <head>
>     <link rel="stylesheet" href="https://unpkg.com/@asyncapi/react-component@1.0.0-next.32/styles/default.min.css">
>     
>     <!-- Custom style sheet -->
>     <link rel="stylesheet" href="./styles/custom.css">
>     
>     <!-- OR as inline style -->
>     <style>
>       html{-moz-tab-size:4;-o-tab-size:4;tab-size:4;line-height:1.15;-webkit-text-size-adjust:100%};
>       body{margin:0;font-family:system-ui,-apple-system,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji};
>     </style>
>   </head>
>   <body>
>     
>     <div id="asyncapi"></div>
> 
>     <script src="https://unpkg.com/@asyncapi/react-component@1.0.0-next.32/browser/standalone/index.js"></script>
>     <script>
>       AsyncApiStandalone.render({
>         schema: {
>           url: 'https://raw.githubusercontent.com/asyncapi/spec/v2.0.0/examples/2.0.0/streetlights.yml',
>           options: { method: "GET", mode: "cors" },
>         },
>         config: {
>           show: {
>             sidebar: true,
>           }
>         },
>       }, document.getElementById('asyncapi'));
>     </script>
> 
>   </body>
> </html>
> ```

Reviewer 2: I think that here we should only focus on the style definition, so please change it to the:

 ```suggestion
If you are using the [standalone bundle](./docs/usage/standalone-bundle.md), you can put the custom styles as a style sheet link or as an inline style in the `<head>` section of the HTML code:

```html
  <head>
    <!-- Custom style sheet -->
    <link rel="stylesheet" href="./styles/custom.css">

    <!-- OR as inline style -->
    <style>
      html{-moz-tab-size:4;-o-tab-size:4;tab-size:4;line-height:1.15;-webkit-text-size-adjust:100%};
      body{margin:0;font-family:system-ui,-apple-system,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji};
    </style>
    
    <link rel="stylesheet" href="https://unpkg.com/@asyncapi/react-component@1.0.0-next.32/styles/default.min.css">
    
    ...
  </head>
```
```

  
### Custom logo

> The AsyncAPI component supports the option to use a custom logo. By using the `x-logo` custom extension in the InfoObject, a logo will be shown in the top left corner.

Reviewer 2: ```suggestion
The AsyncAPI component supports the option to use a custom logo. By using the `x-logo` custom extension in the [InfoObject](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#infoObject), a logo will be shown in the top left corner.

> **NOTE**: The logo will only appear if the [sidebar option](./docs/configuration/config-modification.md#definition) is enabled.
```

  
```yaml
asyncapi: 2.2.0
info:
  title: Account Service
  version: 1.0.0

>   description": This service is in charge of processing user signups

Reviewer 2: ```suggestion
  description: This service is in charge of processing user signups.
```

  x-logo: 'https://raw.githubusercontent.com/asyncapi/spec/master/assets/logo.png'
channels:
  ...
```

> Example:
> ![2022-01-25 at 15 18 23](https://user-images.githubusercontent.com/952446/150993843-8fbf5c90-50ed-4aed-8268-01ce27ecf173.png)  

Reviewer 2: We shouldn't show any picture :) Please remove it.

```suggestion
```

import "@asyncapi/react-component/styles/default.min.css";
```

### Custom styles
The asyncapi react-component does not set any global fonts. This provides the option to define your custom font-family and related font styling.
This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using asyncapi react-component.

Custom font styling (like used in AsyncAPI studio): `styles/custom.css` >>
```css
  html {
  -moz-tab-size: 4;
  -o-tab-size: 4;
  tab-size: 4;
  line-height: 1.15;
  -webkit-text-size-adjust: 100%;
}

body {
  margin: 0;
  font-family: system-ui, -apple-system, Segoe UI, Roboto, Helvetica, Arial, sans-serif, Apple Color Emoji, Segoe UI Emoji;
}
```

If you are using the component in React or another front-end framework, don't forget to import the custom style.
``` js
import "@asyncapi/react-component/styles/default.min.css";
import "styles/custom.css";
```
  

> If you are using the standalone bunlde, you can put the custom font styling as a style sheet link or as an inline style in the `<head>` section of the HTML code.

Reviewer 3: ```suggestion
If you are using the standalone bundle, you can put the custom font styling as a style sheet link or as an inline style in the `<head>` section of the HTML code.
```
Please correct the spelling. 😊
